Applications/DisplayProperties/DisplayProperties.cpp

```diff
@@ -0,0 +1,145 @@
+{
+    auto* tabWidget = new GTabWidget(m_root_widget);
+
+    // First, let's create the "Background" page in the tabbed journal
+    auto* background_splitter = new GSplitter(Orientation::Vertical, nullptr);
+    tabWidget->add_widget("Background", background_splitter);
+
+    auto* background_content = new GWidget(background_splitter);
+    background_content->set_background_color(Color::WarmGray);
+    background_content->set_layout(make<GBoxLayout>(Orientation::Vertical));
+    background_content->layout()->add_spacer();
+    background_content->layout()->set_margins({4, 4, 4, 4});
+
+    auto* wallpaper_list = new GListView(background_content);
+    wallpaper_list->set_background_color(Color::White);
+    wallpaper_list->set_model(*ItemListModel<AK::String>::create(m_wallpapers));
+    wallpaper_list->horizontal_scrollbar().set_visible(false);
+    wallpaper_list->on_selection = [this](auto& index) {
+        m_selected_wallpaper = m_wallpapers.at(index.row());
+    };
+
+    // Let's add the settings tab
+    auto* settings_splitter = new GSplitter(Orientation::Vertical, nullptr);
+    tabWidget->add_widget("Settings", settings_splitter);
+
+    auto* settings_content = new GWidget(settings_splitter);
+    settings_content->set_background_color(Color::WarmGray);
+    settings_content->set_layout(make<GBoxLayout>(Orientation::Vertical));
+    settings_content->layout()->add_spacer();
+    settings_content->layout()->set_margins({4, 4, 4, 4});
+
+    auto* resolution_list = new GListView(settings_content);
+    resolution_list->set_background_color(Color::White);
+    resolution_list->set_model(*ItemListModel<Size>::create(m_resolutions));
+    resolution_list->horizontal_scrollbar().set_visible(false);
+    resolution_list->on_selection = [this](auto& index) {
+        m_selected_res = m_resolutions.at(index.row());
+    };
+
+
+    // Add the apply and cancel buttons
+    auto* bottom_widget = new GWidget(m_root_widget);
+    bottom_widget->set_layout(make<GBoxLayout>(Orientation::Horizontal));
+    bottom_widget->layout()->add_spacer();
+    bottom_widget->set_size_policy(Orientation::Vertical, SizePolicy::Fixed);
+    bottom_widget->set_preferred_size(1, 22);
+
+    auto* apply_button = new GButton(bottom_widget);
+    apply_button->set_text("Apply");
+    apply_button->set_size_policy(Orientation::Vertical, SizePolicy::Fixed);
+    apply_button->set_size_policy(Orientation::Horizontal, SizePolicy::Fixed);
+    apply_button->set_preferred_size(60, 22);
+    apply_button->on_click = [this](GButton&) {
+        apply_settings();
+    };
+
+    auto* cancel_button = new GButton(bottom_widget);
+    cancel_button->set_text("Cancel");
+    cancel_button->set_size_policy(Orientation::Vertical, SizePolicy::Fixed);
+    cancel_button->set_size_policy(Orientation::Horizontal, SizePolicy::Fixed);
+    cancel_button->set_preferred_size(60, 22);
+    cancel_button->on_click = [this](GButton&) {
+        GApplication::the().quit(EXIT_SUCCESS);
```

Comment:
You can omit the `EXIT_SUCCESS` here.
```suggestion
        GApplication::the().quit();
```